README.md

```diff
@@ -8,335 +8,81 @@ C extension for Spiffe platform.
-```
-
-## Building
-Build the c-spiffe project:
-```
-cd /mnt
-mkdir build && cd build
-cmake ..
-make
-make test
-```
-After running `make test`, you will find the test files into `Testing` folder.
-
-### Code Coverage Support
-
-This implements Code Coverage Reports using either using either `gcov` or `lcov`.
-If you want to check them, you should run the following command after `make test`:
-
-```
-make gcov
-make lcov
-```
-
-The coverage reports will be into `Coverage` folder. In the case of `lcov`, you
-can see into the browser, opening the `index.html` file on the folder above.
-
-# Continuous Integration (CI)
-
-## Operation
-
-Continuous integration (CI) is a practice where a team of developers integrate their code early and often to the main branch or code repository. The goal is to reduce the risk of seeing “integration hell” by waiting for the end of a project or a sprint to merge the work of all developers.
-
-To adopt continuous integration, we will need to run your tests on every change that gets pushed back to the main branch. To do so, you will need to have a service that can monitor your repository and listen to new pushes to the codebase. 
-
-![Alt text](img/ci-process.png "Commit, Build and Deploy")
-
-# Introduction
-
-We only need an existing GitHub repository to create and run a GitHub Actions workflow. In this guide, you'll add a workflow that lints multiple coding languages using the GitHub Super-Linter action. The workflow uses Super-Linter to validate your source code every time a new commit is pushed to your repository.
-
-# To use Github Actions CI/CD, we need:
-
-Most CI tools integrate seamlessly with Git services — especially GitHub.
-
-<ol>
-    <li>
-    From your repository on GitHub, create a new file in the .github/workflows directory named superlinter.yml.
-    </li>
-    <li>
-    Copy the following YAML contents into the superlinter.yml file. Note: If your default branch is not main, update the value of DEFAULT_BRANCH to match your repository's default branch name.
-
-```
-name: Super-Linter
-
-# Run this workflow every time a new commit pushed to your repository
-on: push
-
-jobs:
-  # Set the job key. The key is displayed as the job name
-  # when a job name is not provided
-  super-lint:
-    # Name the Job
-    name: Lint code base
-    # Set the type of machine to run on
-    runs-on: ubuntu-latest
+[SPIFFE](https://spiffe.io/) stands for Seurity Identity Framewor for Evertyone and is a set for securely identifying system in dynamic and heteregeous environment. Please refer to https://spiffe.io/docs/latest/spiffe-about/overview/ for more information.  
+C-spiffe is a extension for Spiffe that allows any [workload](https://spiffe.io/docs/latest/spiffe-about/spiffe-concepts/#workload) written C, C++ or even any language witch can import a .so library, access [Workload API](https://spiffe.io/docs/latest/spiffe-about/spiffe-concepts/#spiffe-workload-api) and establish a mTLS connection.  
```

Comment:
written in* C, C++ or any language that supports loading a .so library, access the Workload API and establish mTLS connections with other workloads.